<?xml version="1.0"?>
<launch>

<node name="tracker" pkg="vilib" type="vilib_ros" output="screen">
  <rosparam file="$(find vilib)/params/euroc.yaml"/>
  <remap from="tracker/image" to="/cam0/image_raw"/>
</node>

</launch>